nexosim/src/server/run.rs

```diff
@@ -198,6 +199,7 @@ fn run_local_service(
 struct GrpcSimulationService {
     init_service: Mutex<InitService>,
     controller_service: Arc<Mutex<ControllerService>>,
+    endpoint_service: Mutex<InspectorService>,
```

Comment:
I think this should now be changed to `inspector_service` :-)

Reply:
Oh my, I was to quick with pushing.